modules/dw1000/dw1000.c

```diff
@@ -36,7 +36,14 @@ void dw1000_handle_interrupt(struct dw1000_instance_s* instance) {
     dw1000_write(instance, DW1000_SYSTEM_EVENT_STATUS_REGISTER_FILE, 0, sizeof(sys_status), &sys_status);
 }
 
-void dw1000_init(struct dw1000_instance_s* instance, uint8_t spi_idx, uint32_t select_line, uint32_t reset_line) {
+//call this method after every subtraction of timestamps
+int64_t dw1000_wrap_timestamp(int64_t ts) {
```

Comment:
Does truncating after subtracting not do the same thing?
Some python experimentation:
```
>>> 34-50
-16
>>> (34-50) & 0xff
240
>>> (34-50) + 0xff+1
240
```